com.github.Desktop.yaml

```
@@ -0,0 +1,96 @@
+app-id: com.github.Desktop
+base: org.electronjs.Electron2.BaseApp
+base-version: "20.08"
+runtime: org.freedesktop.Platform
+runtime-version: "20.08"
+sdk: org.freedesktop.Sdk
+sdk-extensions:
+  - org.freedesktop.Sdk.Extension.node14
+command: start-github-desktop
+separate-locales: false
+finish-args:
+  - --share=ipc
+  - --socket=x11
```

```suggestion
  - --socket=fallback-x11
```

This assuming the app supports wayland, otherwise remove the wayland socket.